pkg/assets/builder_test.go

```diff
@@ -19,6 +19,7 @@ package assets
 import (
 	"testing"
 
+	"github.com/pkg/errors"
```

Comment:
Since you pulled in the errors package you'll need to run `hack/update-bazel.sh` and checkin the resulting files.

Reply:
That was actually the wrong auto-complete, good thing it failed